src/model/decorators/DraftDecoratorType.js

```diff
@@ -25,7 +26,7 @@ export type DraftDecoratorType = {
   /**
    * Given a `ContentBlock`, return an immutable List of decorator keys.
    */
-  getDecorations(block: ContentBlock): List<?string>,
+  getDecorations(contentState: ContentState, block: ContentBlock): List<?string>,
```

Comment:
This would require us to also change the call sites for `BlockTree.generate` as those pass in the block which is forwarded to `getDecorations`. That happens while mapping over a content state's blocks. I agree that it would be nice to make sure that the lookup takes place within `getDecorations` using a block key and not a block instance. Having said that, since `getDecorations` is not a method which a consumer would call directly, is it really worth changing?